allcatchr/1.0.0/Dockerfile

```diff
@@ -0,0 +1,52 @@
+############################# Dockerfile ####################################
+FROM biocontainers/biocontainers:latest
+
+############################# METADATA ######################################
+
+LABEL    software="allcatchr" \
+    base_image="biocontainers:latest" \
+    container="allcatchr" \
+    about.summary="The gene expression classifier ALLCatchR identifies B-precursor ALL subtypes and underlying developmental trajectories" \
+    about.home="https://github.com/ThomasBeder/ALLCatchR" \
+    software.version="1.0.0" \
+    version="1" \
+    extra.identifiers.biotools="allcatchr" \
+    about.copyright="2022, Thomas Beder" \
+    about.license="MIT" \
+    about.license_file="https://github.com/ThomasBeder/ALLCatchR/blob/main/LICENSE.txt" \
+    maintainer="Nadine Wolgast <NadineWolgast@gmx.de.br>"\
```

Comment:
invalid extra slash on last line, docker build fails

Reply:
I'm not sure if the installation RUN statement is correct. It is a R package that needs devtools to be installed. Could you suggest me how I can change it so it works?

Reply:
I do not know how R works... so no idea